src/sdgio-edit.owl

```diff
@@ -1515,6 +1519,24 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynony
 AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020232> "system level outcome"@en)
 SubClassOf(<http://purl.unep.org/sdg/SDGIO_00020232> <http://purl.unep.org/sdg/SDGIO_00000251>)
 
+# Class: <http://purl.unep.org/sdg/SDGIO_00020236> (intermediate development outcome)
+
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.unep.org/sdg/SDGIO_00020236> <http://orcid.org/0000-0062-3253-4387>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020236> "2019-04-28T11:19:21Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.unep.org/sdg/SDGIO_00020236> "IDO")
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://hdl.handle.net/10947/3865>) rdfs:comment <http://purl.unep.org/sdg/SDGIO_00020236> "A CGIAR objective specification which 1) names an objective the achievement of which contributes to the achievement of a specific system level outcome and 2) indicates to which system level outcome a piece of CGIAR research contributes and how such a piece of research contributes to that system level outcome.")
```

Comment:
Some definitional points:
* It's a bit strange to say that something is an objective and also names an objective.
* Are the IDOs limited to research activity? I think operational processes fall under this, but perhaps @marieALaporte can provide insight here.
* It seems that the IDO definition is very overloaded: it does far too much for a single objective specification. It can't both direct "what" is to be done and "how" it is to be done. The "how" must be defined in a `plan specification`. @Mttbnchtt: if and IDO is really supposed to do all this, we will have to unpack this concept into both ObjSpec and a PlanSpec.

I'd rephrase this definition to:

```
A CGIAR objective specification which describes the intended endpoint and outcome(s) of a process which, itself, is part of and contributes to a larger process directed by a system level outcome.

Reply:
I like how you rephrase your definition and I will implement it. I will also try to de-load the definition. Here is the main piece of text from which I got the definition: "To ground these SLOs at the level of research activities, CGIAR has introduced the concept of Intermediate Development Outcome (IDOs), which enable researchers to think through the contexts in which their outputs might contribute to development outcomes" (p. 14 of the document titled _CGIAR Strategy and Results Framework_).

On a minor point, I took (and still take) an objective _specification_ to be a linguistic entity whose designatum is an objective. This is why I began my suggested definition as "an objective specification which names etc.". However, your suggestions "objective speciication which describes ..." is surely clearer and I will adopt it.

Reply:
On second thought, I am not sure that an objective specification describes something rather than being a label to refer to something. Like 'point' refers to a point but does not describe a point (or the concept of point). Therefore, I will proceed to de-load the definition but, for now, I will not say that an objective specification describes something. If you confirm your suggestion of saying that an objective specification describes something, I will do so as soon as I read your new comment.

(Also let me know if I am just being picky in an unhelpful way or observations of this type are useful. Building an ontology is partly an art and I am trying to understand your way of doing it.)

Reply:
For now I have modified the definition of IDO as follows:
A CGIAR objective specification which names an objective the achievement of which contributes to the achievement of a specific system level outcome.

Moreover, I modified the definition of sub-IDO as follows:
A CGIAR objective specification which names an objective the achievement of which contributes to the achievement of an intermediate development outcome.

As I said, I am pausing on your suggestion to say that an IDO describes something. However, I am pausing only because I would like to hear from you more about this point. I will adopt your suggestion if you confirm it.

Reply:
To make matters more complicated, the json file does call IDOs descriptions. Here is a snippet:

  {
    "code": "1",
    "description": "Increased resilience of the poor to climate change and other shocks",
    "isCrossCutting": false,
    "srfCrossCuttingIssue": null
  }

So maybe your suggestion is right (because a description describes something). Let me know what you think is appropriate. Let me also know if it is inappropriate for me to do all this thinking about a single issue that is probably a minor one.

Reply:
In the end, I accepted Pier Luigi's suggestion, as I understood that he repeated it on Skype. We will talk about this on 29 Apr 2019.

Reply:
> To make matters more complicated, the json file does call IDOs descriptions. Here is a snippet:

I think that in the json output when it says description it should be just saying label or something similar. No real description are really given. You can see the same pattern for other calls. Let me confirm this and come back to you @Mttbnchtt

Reply:
> Are the IDOs limited to research activity? I think operational processes fall under this, but perhaps @marieALaporte can provide insight here.

@pbuttigieg I am going to look into that too as I am not sure about it

Reply:
@Mttbnchtt: we use "describes" to align with IAO: 
http://purl.obolibrary.org/obo/IAO_0000005

This is where conversations on the metaphysics should go (e.g. see https://github.com/information-artifact-ontology/IAO/issues/213). They are useful, but not likely to be consequential in our application scenario and will just slow things down. We can realign if there's a need to later or if there's a philosophical breakthrough (which can take far too long for our use case). 
>>To make matters more complicated, the json file does call IDOs descriptions. Here is a snippet:
>
> I think that in the json output when it says description it should be just saying label or something similar. No real description are really given.

Yes, this looks like something that's just a label for the format and nothing that should be thought about too ontologically.

Reply:
got it

Reply:
I am going to resolve this conversation.
Summary of the issue: the definition of the IDO class was too complicated and did not align with conceptual rules states in IAO.
Solution: I modified the definition according to Pier Luigi's suggestion.